src/Components/Onboarding/Components/OnboardingMarketingCollection.tsx

```diff
@@ -21,17 +20,6 @@ const OnboardingMarketingCollection: FC<OnboardingMarketingCollectionProps> = ({
   description,
 }) => {
   const artworks = extractNodes(marketingCollection.artworks)
-  const { onClose } = useOnboardingContext()
-  const tracking = useOnboardingTracking()
-
-  useEffect(() => {
-    return () => {
-      if (onClose) {
-        tracking.userCompletedOnboarding()
-        onClose()
-      }
-    }
-  }, [onClose, tracking])
```

Comment:
Previously, the component would trigger both closure and tracking events immediately upon re-render.